semopenalex-app/data/templates/http%3A%2F%2Fwww.metaphactory.semopenalex.com%2Fapp%2FStart.html

```diff
@@ -122,27 +122,27 @@
         {
         	"iri": "<http://www.w3.org/2004/02/skos/core#Concept>",
           "label": "Topic",
-          "thumbnail": "/assets/images/lightbulb.png"
+          "thumbnail": "/assets/images/icon-5.png"
```

Comment:
please add a comment on top of the categories block stating that the thumbnails are used for "ordering" the domains / cards

Reply:
done.